packages/next-server/server/next-server.js

```diff
@@ -174,9 +178,6 @@ export default class Server {
     }
 
     const html = await this.renderToHTML(req, res, pathname, query)
-    if (isResSent(res)) {
```

Comment:
This should definitely be here.

Reply:
If I'm not wrong `undefined` is returned by `this.renderToHTML()` in that case, can this be changed to

```js
if (html === undefined) {
  return;
}
```